pstoreds/ds_test.go

```diff
@@ -0,0 +1,69 @@
+package pstoreds
```

Comment:
i know it's super long but i think `peerstoreds` might be best here. what do you think

Reply:
I've noticed that idiomatic Go uses abbreviations for longer terms. I had set on `pstore` for `peerstore` Don't you think `peerstoreds` is somewhat long and pedantic, though? e.g. we don't use `multiaddrnet` but `manet`. Also, the term peerstore is already implicit from the import path.

Anyway, I'm not married to either name ;-)